_source/api/logzio-public-api.yml

```diff
@@ -6046,7 +6129,7 @@ definitions:
         $ref: '#/definitions/Pagination'
 
 
-  # ::::: Alert
+  # ::::: Alert DEFINITIONS
```

Comment:
was wondering why the alert section is on this pr

Reply:
Unrelated - added to make it easier to find in 7000+ lines of text